src/sentry/migrations/0177_discover_time_to_timestamp_to_hour.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 1.11.29 on 2021-03-12 12:38
+
+from django.db import migrations
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def time_to_timestamp_to_hour(apps, schema_editor):
+    DiscoverSavedQuery = apps.get_model("sentry", "DiscoverSavedQuery")
+    for query in RangeQuerySetWrapperWithProgressBar(DiscoverSavedQuery.objects.all()):
+        fields = query.query["fields"]
+        if "time" in fields:
```

Comment:
Is "time" used in any query conditions (ie. `query.query["query"]`) that we need to update as well?

Reply:
Hm, somehow I totally forgot about query query query. I don't think anyone on sentry.io is using it but we should migrate it for self-hosted anyways.

Reply:
There are some users on sentry.io, but writing a migration is complicated. Additionally, "time" will continue to work as before in queries, so migrating it isn't mandatory.